src/cli/utils/onedrive_test.go

```diff
@@ -4,12 +4,21 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/alcionai/corso/src/cli/utils"
 	"github.com/alcionai/corso/src/pkg/selectors"
 )
 
-func (suite *ExchangeUtilsSuite) TestIncludeOneDriveRestoreDataSelectors() {
+type OneDriveUtilsSuite struct {
+	suite.Suite
+}
```

Comment:
What are we gaining by adding this suite?

Reply:
Clarity about the set of tests being run, an alignment with standards elsewhere in the code where in all other cases (as far as I'm aware) suites are localized to a file, and a reduction in potential confusion about why onedrive tests are running as part of an exchange test suite.